### ISSUE TITLE
missing CHANGES.txt in the egg...

### ISSUE THREAD

**Issue by goschtl** [zopefoundation/grokproject#9]
Hey, i just tried to download & install the new version of grokproject. I get this error

Processing grokproject-2.11.tar.gz
Writing /var/folders/00/trkhg3rj2038cznzmdmr5cx80000gn/T/easy_install-EOGcsf/grokproject-2.11/setup.cfg
Running grokproject-2.11/setup.py -q bdist_egg --dist-dir /var/folders/00/trkhg3rj2038cznzmdmr5cx80000gn/T/easy_install-EOGcsf/grokproject-2.11/egg-dist-tmp-EKWoqg
error: [Errno 2] No such file or directory: 'CHANGES.txt'

The message is clear there is a missing CHANGES.txt. I verified this by downloading the and unzipping the package. The CHANGES.txt is indeed missing.

I suggest we have to write a MANIFEST.in file for it? What do you think?

**Comment by prsephton** [comment]
Hi;  You are indeed correct.  I have just tried pip install grokproject myself and met with the same spectacular failure!

The problem seems to be that as you say, CHANGES.txt was not included in the build, and setup.py reads this file to make a "long description".

We can drop @janwijbrand a line and see if he can help;  I don't think its anything too serious.

It strange as I'm pretty sure I tested the cheeseshop release of 2.11 previously.
